Smart Village App/views/content/widgets/wallCreateContentForm.php

```diff
@@ -0,0 +1,134 @@
+                    'preview' => '#contentFormFiles_preview',
+                    'dropZone' => '#contentFormBody',
+                    'max' => Yii::$app->getModule('content')->maxAttachedFiles
+                ]);
+                ?>
+                <?= FileHandlerButtonDropdown::widget(['primaryButton' => $uploadButton, 'handlers' => $fileHandlers, 'cssButtonClass' => 'btn-default']); ?>
+
+                <!-- public checkbox -->
+                <?= Html::checkbox("visibility", "", ['id' => 'contentForm_visibility', 'class' => 'contentForm hidden', 'aria-hidden' => 'true', 'title' => Yii::t('ContentModule.base', 'Content visibility') ]); ?>
+
+                <!-- content sharing -->
+                <div class="pull-right">
+
+                    <span class="label label-info label-public hidden"><?= Yii::t('ContentModule.base', 'Public'); ?></span>
+
+                    <ul class="nav nav-pills preferences" style="right: 0; top: 5px;">
+                        <li class="dropdown">
+                            <a class="dropdown-toggle" style="padding: 5px 10px;" data-toggle="dropdown" href="#" aria-label="<?= Yii::t('base', 'Toggle post menu'); ?>" aria-haspopup="true">
+                                <?= Icon::get('cogs')?>
+                            </a>
+                            <ul class="dropdown-menu pull-right">
+                                <li>
+                                    <?= Link::withAction(Yii::t('ContentModule.base', 'Notify members'), 'notifyUser')->icon('bell')?>
+                                </li>
+                                <?php if(TopicPicker::showTopicPicker($contentContainer)) : ?>
+                                    <li>
+                                        <?= Link::withAction(Yii::t('ContentModule.base', 'Topics'), 'setTopics')->icon(Yii::$app->getModule('topic')->icon) ?>
+                                    </li>
+                                <?php endif; ?>
+                                <?php if ($canSwitchVisibility): ?>
+                                    <li>
+                                        <?= Link::withAction(Yii::t('ContentModule.base', 'Change to "Public"'), 'changeVisibility')
+                                            ->id('contentForm_visibility_entry')->icon('unlock') ?>
+                                    </li>
+                                <?php endif; ?>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <?= UploadProgress::widget(['id' => 'contentFormFiles_progress']) ?>
+            <?= FilePreview::widget(['id' => 'contentFormFiles_preview', 'edit' => true, 'options' => ['style' => 'margin-top:10px;']]); ?>
+
+        </div>
+        <!-- /contentForm_Options -->
+        <?= Html::endForm(); ?>
+    </div>
+    <!-- /panel body -->
+</div> <!-- /panel -->
+
+<script>
+    let element = document.getElementById("post_section");
```

Comment:
all right, i found `<div class="panel panel-default clearfix" id="post_section">` in line 39 of this file. this should work fine.